app/confirmation/get.njk

```diff
@@ -37,6 +37,17 @@
                 }
               ],
               [
+                {
+                  text: "Email"
+                },
+                {
+                  text: accountEmailAddress,
+                  attributes: {
+                    id: "account-email-address"
+                  }
+                }
+              ],
```

Comment:
Does it make sense for the order we display the information back to be different from the order the user enters it in? Maybe a question for Conor

Reply:
I'll ask Conner, I put them next to each other since they seemed closely related name/email then sort code/account number next to each other. I'll merge and will update if Conner thinks otherwise, thanks 👍